cirq-core/cirq/experiments/xeb_fitting.py

```diff
@@ -163,10 +161,10 @@ def phased_fsim_angles_from_gate(gate: 'cirq.Gate') -> Dict[str, float]:
         'gamma_default': 0.0,
         'phi_default': 0.0,
     }
-    if gate == SQRT_ISWAP:
+    if gate == ops.SQRT_ISWAP:
         defaults['theta_default'] = -np.pi / 4
         return defaults
-    if gate == SQRT_ISWAP ** -1:
+    if gate == ops.SQRT_ISWAP ** -1:
```

Comment:
@tanujkhattar , Should I change these to `cirq.SQRT_ISWAP_INV` or `ops.SQRT_ISWAP_INV` as well?

Reply:
Yes, this should also change to `ops.SQRT_ISWAP_INV`.